### PR TITLE
add content best practice less modular

### PR DESCRIPTION
**Pull request by mrtuvn**
update

## Purpose of this pull request

This pull request (PR) improve content add approach less best practices modularity

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/theme-best-practice.html

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/theme-best-practice.md [hunk 1]
```diff
@@ -35,5 +35,56 @@ We recommend using the following best practices when developing themes:
...
+
+   //
+   //  Tools, mixins, reused elements
+   //  _____________________________________________
+
+   @import 'theme/mixins/_mixins.less';
+
+   //  _____________________________________________
+   //
+   //  Custom style for component
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/theme-best-practice.md:66]:
```suggestion
   //  Custom style for components
```

File: src/guides/v2.3/frontend-dev-guide/theme-best-practice.md [hunk 2]
```diff
@@ -35,5 +35,56 @@ We recommend using the following best practices when developing themes:
 1. Magento has a set of [coding standards]({{ page.baseurl }}/coding-standards/bk-coding-standards.html) for both back-end and front-end technologies. Refer to them when needed.
 1. Do not repeat work while styling. Instead, create a class or mixin and call them when needed.
 1. While styling any custom module, add the styling within the module, instead of adding it to the design theme. This way, the style will not be loaded unless the module is called. For example `app/code/Company/Module/view/frontend/web/css/source/_module.less`.
+1. While styling custom theme, add styles to seperate less files, instead of adding it to a single file. This way, styles are easier to track down and debug. For example, here is a `app/design/frontend/Company/ThemeCustom/web/css/style-l.less` with styles for desktop.
+
+   **In file style-l.less:**
+   
+   ```less
+   // Define default styles from parent
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/theme-best-practice.md:43]:
```suggestion
   // Define default styles from parent
   ...
```

File: src/guides/v2.3/frontend-dev-guide/theme-best-practice.md [hunk 3]
```diff
@@ -35,5 +35,56 @@ We recommend using the following best practices when developing themes:
...
+
+   @import 'theme/pages/_homepage.less';
+   @import 'theme/pages/_login.less';
+   @import 'theme/pages/_product-details.less';
+   @import 'theme/pages/_account.less';
+   @import 'theme/pages/_contact.less';
+   @import 'theme/pages/_category.less';
+   @import 'theme/pages/_cart.less';
+   @import 'theme/pages/_checkout.less';
+   ```
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/theme-best-practice.md:89]: I would suggest adding a short conclusion here. Something like. 

> As you can see, styles for every component as well as for every page are declared in a separate file.

**Comment by dobooth** [comment]
running tests

**Pull request merged**